content/applications/general/integrations/cloud_storage.rst

```diff
@@ -0,0 +1,271 @@
+=============
+Cloud storage
```

Comment:
NIT: Cloud Storage

Be consistent with other names e.g. `Google Translate` `Barcode Lookup`